qualification/baseline_correlation_products/test_baselines.py

```diff
@@ -41,19 +41,18 @@ async def test_baseline_correlation_products(
     expected output appears in the correct baseline product.
     """
     receiver = receive_baseline_correlation_products  # Just to reduce typing
-    pdf_report.step("Connect to CBF's product controller to retrieve configuration for the running CBF.")
```

Comment:
Not a fan of this log message? 😄

Reply:
It didn't really match the code: the code below doesn't create a connection or retrieve anything.